client/src/pages/Projects/index.tsx

```diff
@@ -1,24 +1,124 @@
+        <div className="basis-7/12 flex flex-row grow items-center">
+          <div className="m-2.5 p-2.5 basis-3/12">
+            {renderProjectLogos(projectData.logos)}
+          </div>
+          <div className="m-2.5 p-2.5 basis-9/12 grow md:border-r">
+            {projectData.descs.descShort}
+          </div>
+        </div>
+
+        <div className="basis-5/12 flex flex-row grow items-center">
+          <div className="m-2.5 p-2.5 basis-1/4 ">
+            <span>Token:</span>
+            <div className="font-bold">{projectData.token.token}</div>
+          </div>
+          <div className="m-2.5 p-2.5 basis-1/4 ">
+            {/*<span style={{ whiteSpace: "nowrap"}}>Total Supply:</span>*/}
+            <span className="truncate">Total Supply:</span>
+            <div className="font-bold text-sm">
+              {projectData.token.totalSupply?.toLocaleString("en-US") ?? "?"}
+            </div>
+          </div>
+          <div className="m-2.5 p-2.5 basis-2/4 text-2xl">
+            {renderProjectURLs(projectData.urls)}
+          </div>
+        </div>
+      </div>
+    );
+  };
+
+  const renderProjectLogos = (projectLogos: ProjectLogos) => {
+    let logo = projectLogos.logoDefault;
+    let compactLogo = projectLogos.logoCompact ?? projectLogos.logoDefault;
+    if (theme === Themes.dark) {
+      if (projectLogos.logoDark) logo = projectLogos.logoDark;
+      if (projectLogos.logoCompactDark)
+        compactLogo = projectLogos.logoCompactDark;
+    }
+    return (
+      <div>
+        <img src={logo} className="logo hidden sm:block"></img>
+        <img src={compactLogo} className="logo sm:hidden"></img>
+      </div>
+    );
+  };
+
+  const renderProjectURLs = (projectURLs: ProjectURLs) => {
+    var elements: JSX.Element[] = [];
+    if (projectURLs.medium) {
+      elements.push(
+        <a href={projectURLs.medium} className="text-medium m-1">
+          <FontAwesomeIcon icon={faMedium} />
+        </a>
+      );
+    }
+    if (projectURLs.twitter) {
+      elements.push(
+        <a href={projectURLs.twitter} className="text-twitter m-1">
+          <FontAwesomeIcon icon={faTwitter} />
+        </a>
+      );
+    }
+    if (projectURLs.discord) {
+      elements.push(
+        <a href={projectURLs.discord} className="text-discord m-1">
+          <FontAwesomeIcon icon={faDiscord} />
+        </a>
+      );
+    }
+    if (projectURLs.telegram) {
+      elements.push(
+        <a href={projectURLs.telegram} className="text-telegram m-1">
+          <FontAwesomeIcon icon={faTelegram} />
+        </a>
+      );
+    }
+    /*if (projectURLs.cardanoScan) {
+      elements.push(<a href={projectURLs.cardanoScan}><img src=""/></a>);
+    }
+    if (projectURLs.poolpm) {
+      elements.push(<a href={projectURLs.poolpm}><img src=""/></a>);
+    }*/
+    return elements;
+  };
+
   return (
     <Page>
       <div className="text-3xl">
         Explore TosiDrop Projects
         <ul>
```

Comment:
it is better to use flexbox for space between elements. You can do
```
<div className="flex flex-col gap-4">
  { map function }
</div>
```
using this you don't need to manually add margin to each box, hence less place to change, when needed.

the standard for flex gap is :
- `gap-2`: for text, e.g., space between icon and text
- `gap-4`: space between bigger component (between buttons etc.)
- `gap-8`: space between title and page